src/lib/validate/validateArgs.js

```diff
@@ -0,0 +1,285 @@
+const _ = require('lodash');
+
+/**
+ * except for checking args and inputs,
+ * this function also check canary policy and parse it into variable policy and return
+ * @param logger
+ * @param serviceName
+ * @param functionName
+ * @param inputs
+ * @param args
+ * @param fcHelper
+ * @returns {Promise<*>}
+ */
+async function validateParams(
+  logger,
+  serviceName,
+  functionName,
+  inputs,
+  args,
+  fcHelper,
+) {
+
+
+  if (serviceName == undefined) {
```

Comment:
这种校验没啥意义吧，因为serviceName是作为函数参数传递的，举个例子：validateParams(mylog, 'demo-function')，也就是将functionName作为serviceName传递了，此时校验是通过的，但实际上是有问题的。
如果要做强校验，函数参数可以定义成一个object，例如：
```suggestion
  async function validateParams(params) {
      const { logger, serviceName, functionName } = params
      if (serviceName) {}
  }
```

Reply:
学习到了

Reply:
fixed